packages/CodeQuality/tests/Rector/If_/SimplifyIfReturnBoolRector/Fixture/should_keep_comments.php.inc

```diff
@@ -0,0 +1,65 @@
+<?php
+
+namespace Rector\CodeQuality\Tests\Rector\If_\SimplifyIfReturnBoolRector\Fixture;
+
+final class ShouldKeepComments
+{
+    public function before($var): bool
+    {
+        // Imagine this is a very important comment here that should not be removed.
+        if (! is_string($var)) {
+            return false;
+        }
+
+        return true;
+    }
+
+    public function after($var): bool
+    {
+        if (! is_string($var)) {
+            return false;
+        }
+
+        // Imagine this is a very important comment here that should not be removed.
+        return true;
+    }
+
+    public function both($var): bool
+    {
+        // Imagine this is a very important comment here that should not be removed.
+        if (! is_string($var)) {
+            return false;
+        }
+
+        // Imagine this is another very important comment here that should not be removed.
+        return true;
+    }
+}
+?>
+-----
+<?php
+
+namespace Rector\CodeQuality\Tests\Rector\If_\SimplifyIfReturnBoolRector\Fixture;
+
+final class ShouldKeepComments
+{
+    public function before($var): bool
+    {
+        // Imagine this is a very important comment here that should not be removed.
+        return is_string($var);
+    }
+
+    public function after($var): bool
+    {
+        // Imagine this is a very important comment here that should not be removed.
+        return is_string($var);
+    }
+
+    public function both($var): bool
+    {
+        // Imagine this is a very important comment here that should not be removed.
+        // Imagine this is another very important comment here that should not be removed.
+        return is_string($var);
+    }
+}
+?>
```

Comment:
If the code should remain the same, the part under `-----` can be fully removed here

Reply:
It's not the same : the Rector changed the if conditions. Only the comments are the same, but I can't use a "one part" fixture file then.

Reply:
Ah, I see.